docs/contributing/templates/c2c-source.md

```diff
@@ -0,0 +1,103 @@
+---
+id: c2c-source  #example: druva-source
+title: XYZ Source #example: Druva Source
+sidebar_label: XYZ #example: Druva
+description: Description goes here. #example: Learn how to configure the Druva Cloud-to-Cloud Source in your Sumo Logic environment.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+{{source icon}}
+
```

Comment:
Can we add a place holder for introduction about the C2C before mentioning the data types.

Reply:
That's underneath this (line 16)